tracee-ebpf/main.go

```diff
@@ -285,6 +287,8 @@ Use this flag multiple times to choose multiple capture options
 			if len(outDir) == 0 {
 				return tracee.CaptureConfig{}, fmt.Errorf("capture output dir cannot be empty")
 			}
+		} else if cap == "profile" {
+			capture.Profile = true
```

Comment:
If I understand correctly, exec is required for profile, and profile without exec is non-functional (correct me if I'm wrong). In this case we should automatically set `capture.Exec=true` here as well.

Reply:
OK

Reply:
https://github.com/aquasecurity/tracee/pull/725/commits/718b24868f23b657de987413eebf28c956137c67